frontend/src/Components/Map/Inputs/Hamburger.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import '../../../css/Sidebar.css';
+import { getAllSurveyData } from '../../../queries/conditions';
+//import { ISurvey } from '../../../backend/src/models.ts';
+
+type Survey = {
+  id: string;
+  timestamp: string;
+};
```

Comment:
```suggestion
export type SurveyList = {
  id: string;
  timestamp: string;
}[];
```
and put it in the models/models.ts

Reply:
<img width="444" alt="image" src="https://github.com/DTU-SE-Group-D/LiRA-Viz/assets/45044551/7804e2ab-e7a5-466c-b610-21eca8e22cf3">

Done.